policy/predicate/signature.go

```diff
@@ -59,33 +73,57 @@ type HasValidSignaturesBy struct {
 
 var _ Predicate = &HasValidSignaturesBy{}
 
-func (pred *HasValidSignaturesBy) Evaluate(ctx context.Context, prctx pull.Context) (bool, string, error) {
+func (pred *HasValidSignaturesBy) Evaluate(ctx context.Context, prctx pull.Context) (*common.PredicateResult, error) {
 	commits, err := prctx.Commits()
+
+	predicateResult := common.PredicateResult{
+		Satisfied:       false,
+		ConditionPhrase: "have valid signatures by",
```

Comment:
```suggestion
		ConditionPhrase: "have valid signatures by members of",
```

I think this also only applies in the commit hash case. For the signer case, a better phrase might be `meet the membership conditions`